### DISCUSSION TITLE
Data from BirdNet not uploading to Birdweather

### DISCUSSION THREAD

**Discussion by skyjoe66** [mcguirepr89/BirdNET-Pi#1209] [General]
I recently installed birdnetpi and have been successful in recording and identifying some local birds here in North Central Florida.  

I registered 2 stations - 4592  and 4602. I entered that station ID in my Birdnet-Pi settings.

But, my station data fails to appear on Birdweather web site and when I check the logs, I see this error

10:43:11---[utils.reporting][INFO] Soundscape POST Response Status - 403
10:43:11---[utils.reporting][ERROR] Access denied

It seems like I would need to add my weatherbird API Token somewhere, but I do not see a field for that. 

Any suggestion on how to get my data to Birdweather?

**Comment by jalbarra** [comment]
Hi, you need to look for in "Settings"

**Reply by skyjoe66** [reply]
I have my settings set up as in my post.  

I registered 2 stations - 4592 and 4602. I entered that station ID in my Birdnet-Pi settings. I do not see anywhere to set up my token.

**Comment by skyjoe66** [comment]
SOLVED: In the instructions on BirdNet, settings direct the user to enter a Birdweather ID. However, when registering your station on app.birdweather.com/account/stations, there is no Birdweather ID presented.  You are given a username (email address) and a station ID, and a Birdweather Token.  The Token is your birdweather ID.  Also, be patient after you enter your Birdweather token as the script that updates birdweather will only update if high probability detections are recorded.

**Reply by HasseM** [reply]
Thanks! |
**Token = birdweather ID**
